api/src/services/authentication.ts

```diff
@@ -285,6 +285,7 @@ export class AuthenticationService {
 		const record = await this.knex
 			.select({
 				session_expires: 's.expires',
+				session_next: 's.next_token',
```

Comment:
Rename for consistency.

```suggestion
				session_next_token: 's.next_token',
```